lib/parse.js

```diff
@@ -520,6 +522,11 @@ function tokenizer($TEXT, filename, html5_comments, shebang) {
         if (RE_OCT_NUMBER.test(num) && next_token.has_directive("use strict")) {
             parse_error("Legacy octal literals are not allowed in strict mode");
         }
+        if (num.endsWith("n")) {
+            if (!has_dot && RE_BIG_INT.test(num))
+                return token("bigInt", num.replace("n", ""));
```

Comment:
Please change the token type to `bigint`. All token types have all-lowercase names.

Reply:
Sure, will do it soon